common/changes/@itwin/core-frontend/geo-fix_hanging_maplayer_test_2023-11-29-22-04.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-frontend",
+      "comment": "Fix intermittently hanging map layer test",
```

```suggestion
      "comment": "",
```

these change files are supposed to be user facing